uc-crux-llvm/README.md

```diff
@@ -0,0 +1,267 @@
+# UC-Crux-LLVM
+
+UC-Crux-LLVM is a tool for under-constrained symbolic execution of C programs.
+It can be used to find undefined behavior and failing assertions, or for simple
+functions, to formally verify the absence of such behaviors.
+
+**UC-Crux-LLVM is still in development.**
+
+## Demo
+
+<!-- NOTE(lb) These programs are in the test suite as double_free.c and not_double_free.c --->
+
+<!-- TODO(lb) The printouts from the tool leave a lot to be desired here... --->
+
+`uc-crux-llvm` can use symbolic execution to find the potential double-free in this program:
+```c
+#include <stdlib.h>
+void double_free(int* ptr, int x) {
+  if (x % 2 == 0) {
+    free(ptr);
+  }
+  if (x % 3 == 0) {
+    free(ptr);
+  }
+}
+```
+```
+$ uc-crux-llvm --entry-points double_free double_free.c
+[CLANG] clang "-c" "-DCRUCIBLE" "-emit-llvm" "-g" "-I" "test/programs" "-I" "/home/langston/code/crucible/crux-llvm/./c-src/includes" "-O1" "-o" "crux-build/double_free.bc" "test/programs/double_free.c"
```

Comment:
You might want to scrub personal info from the general README.

Reply:
For sure! Thanks for the catch.